be/src/formats/parquet/group_reader.h

```diff
@@ -121,11 +118,11 @@ class GroupReader {
     // conjunct ctxs that eval after chunk is dict decoded
     std::vector<ExprContext*> _left_conjunct_ctxs;
 
-    // dict filter column
-    std::vector<GroupReaderParam::Column> _dict_filter_columns;
-    // direct read conlumns
-    std::vector<GroupReaderParam::Column> _direct_read_columns;
-
+    // dict filter column index
```

Comment:
Please create a class to encapsulate dict filter column information. because we don't want it to be visible everywhere